recipes/julia/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: julia
+  version: "0.5.2"
+
+source:
+  fn: v0.5.2.tar.gz
+  url: https://github.com/JuliaLang/julia/archive/v0.5.2.tar.gz
+  sha256: efa3303d63e39bf3ea06e23ef68807cca0095c7e0507bdf26eae1d5dee7868c7
+  patches:
+  - rpath.patch
+
+build:
+  skip: True  # [py3k or osx or win]
+  rpaths:
+    - lib/julia/
+  number: 0
+
+requirements:
+  build:
+    - gcc  # [linux]
+    - llvm  # [osx]
+    - cmake
+    - zlib 1.2.8
+    - python
+    - autoconf 2.69 pl5.*
+    - automake 1.15 pl5.*
+    - openblas 0.2.19|0.2.19.*
+    - llvmdev 3.9.*
+    - fftw
+    - gmp 6.1.*
+    - mpfr 3.1.*
+    - curl
+    - libssh2 1.7.*
+    - libxml2 2.9.*
+    - libgit2
+    
+  run:
+    - zlib 1.2.8
+    - libgcc  # [linux]
+    - libgit2
+    - gmp 6.1.*
+    - openblas 0.2.19|0.2.19.*
+
+test:
+  commands:
+    - julia -v
```

Comment:
Should the test be run from both the `build.sh` script and from the `meta.yaml` file, or just one of those?